### DISCUSSION TITLE
Protected labels for PR's and Issues

### DISCUSSION THREAD

**Discussion by rashelle-cld** [community/community#51318] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Github allows for years not, setting labels for PR's and Issues. Meaningful labels allows us to maintain better our repositories, to scope our work and reflect them on milestones.

We now use labels to trigger actions as well, but I'm missing on a key behaviour - protecting labels.
I would love to have protected rules for labels, as I have for branches and tags. 

For example, auto added labels to PR's from the repo's owners (maintenance) should be fixed without an option of being removed by a user.

Just to add, github's discussions create labels automatically without allowing editing/ removing them. 
I find this feature request super handy, especially since we trigger workflows many times for CI pipelines.

Tnx in advance :)

**Comment by alvaromartmart** [comment]
+1, this would be great

**Comment by oleksandr-kinship** [comment]
+1.
